nfc/libs/NfcCoreLib/lib/Nci/phNciNfc.c

```diff
@@ -264,7 +408,6 @@ phNciNfc_ReInitialise(void* pNciHandle,
         pNciContext->ResetInfo.ResetTypeReq = phNciNfc_ResetType_KeepConfig;
         /* Init payload to display the build number in the init response */
         pNciContext->tInitInfo.bExtension = 0x00;
-        pNciContext->tInitInfo.bSkipRegisterAllNtfs = 0x01;
```

Comment:
From best I can tell, the `bSkipRegisterAllNtfs` flag exists so that if `phNciNfc_ReInitialise` is called we don't have to waste time re-registering all the existing notifications. (Granted `phNciNfc_ReInitialise` isn't being used at the moment. So your change technically can't break anything.)